### DISCUSSION TITLE
Day 3: Want to build your own lab manual?

### DISCUSSION THREAD

**Discussion by Kourtney-Burger** [SAEL-SWFSC/PAM-SI-Mtg-March2024#58] [Polls]
Are you interested in using GitHub/Quarto to build your own lab manual like [SWFSC SAEL Lab](https://sael-swfsc.github.io/SAEL-lab-manual/) and [Faylab](https://thefaylab.github.io/lab-manual/)?

**Comment by sofievanparijs** [comment]
This would be great and super useful for us.
